crates/wasm-mutate/src/mutators/rename_export.rs

```diff
@@ -4,19 +4,23 @@ use rand::prelude::SmallRng;
 use rand::{Rng, RngCore};
 use wasm_encoder::{CodeSection, Export, ExportSection, Function, Instruction, Module};
 use wasmparser::{CodeSectionReader, ExportSectionReader};
+
+/// RenameExportMutator generates a random renaming of prexisting exports.
+/// The export entry is selected randmonly and then a new `field` name is generated/
 pub struct RenameExportMutator {
+    /// The maximum length of the generated export entry
     pub max_name_size: u32,
 }
 
 impl RenameExportMutator {
     // Copied and transformed from wasm-smith name generation
-    fn limited_string(&self, rnd: &mut SmallRng) -> String {
-        let size = rnd.gen_range(1, self.max_name_size);
-        let size = std::cmp::min(size, self.max_name_size);
+    fn limited_string(&self, rnd: &mut SmallRng, info: &ModuleInfo, max_name_size: u32) -> String {
+        let size = rnd.gen_range(1, max_name_size);
+        let size = std::cmp::min(size, max_name_size);
         let mut str = vec![0u8; size as usize];
         rnd.fill_bytes(&mut str);
```

Comment:
Unrelated to this PR, but I am just noticing this now: this should call the configured `raw_mutate_func`, if any, and only fallback to using the RNG if there is no `raw_mutate_func`.

Reply:
Done